### PR TITLE
Type attribute example

### PR DESCRIPTION
**Pull request by nwtn**

### CODE REVIEW

**Comment by yoavweiss** [comment]
Maybe switch the JPEG-XR example to JP2K, which has an official IANA mime type? I'm not comfortable with JPXR's lack of official mime type, and the one used looks hacky...

@zcorpan - thoughts?

**Comment by nwtn** [comment]
MS lists that JPEG XR mime type @ http://msdn.microsoft.com/en-us/library/aa359736(v=vs.85).aspx but I'm happy to keep it out if you guys think that makes sense. LMK.

**Comment by yoavweiss** [comment]
OK, in that case, let's keep it in.

File: index.bs [hunk 1]
```diff
@@ -257,6 +257,28 @@ Examples of usage</h3>
...
+
+		<pre>
+				&lt;picture>
+					&lt;source type="image/webp" srcset="dogs-1.webp, dogs-2.webp 2x">
+					&lt;source type="image/vnd.ms-photo" srcset="dogs-1.jxr, dogs-2.jxr 2x">
+					&lt;img src="dogs-1.jpg" srcset="dogs-2.jpg 2x" alt="Hundreds of hot dogs" width="600" height="200">
+				&lt;/picture>
+		</pre>
+
+		In this example, the user agent will chose the first source that has a
```
Comment (yoavweiss) [inline index.bs]: s/chose/choose/ ?

**Comment by yoavweiss** [comment]
Awesome! Thanks! :)

**Pull request merged**